lms/djangoapps/grades/tests/test_signals.py

```diff
@@ -1,7 +1,7 @@
 """
 Tests for the score change signals defined in the courseware models module.
 """
-
+import itertools
```

Comment:
Every day I get to read a review including `import itertools` is a good day.

Reply:
https://www.youtube.com/watch?v=LV9BXdets_8